### DISCUSSION TITLE
Is there an easy way to support files without suffix?

### DISCUSSION THREAD

**Discussion by sbaechler** [react-dropzone/react-dropzone#1261] [Q&A]
Currently we allow files with no suffix. So far we used `''` in the accept array which worked. With the new version there is an error.

Do I have to use a custom validator?

**Comment by rolandjitsu** [comment]
@sbaechler when you say "no suffix", do you mean no extension? A custom validator will probably be the way to go.

**Reply by sbaechler** [reply]
Yes, no file extension. I think the use-case was that a popular erp software created an export file without an extension.
